contracts/StakedCelo.sol

```diff
@@ -46,4 +67,72 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
     function burn(address from, uint256 amount) external onlyManager {
         _burn(from, amount);
     }
+
+    /**
+     * @notice Locks stCELO from an address.
+     * @param account The address that will have its stCELO balance locked.
+     * @param amount The amount of stCELO to lock.
+     */
+    function lockBalance(address account, uint256 amount) external onlyManager {
+        uint256 lockedBalance = _lockedBalances[account];
+        if (lockedBalance < amount) {
+            _lockedBalances[account] = amount;
+            uint256 amountToBurn = amount - lockedBalance;
+            totalLocked += amountToBurn;
+            _burn(account, amountToBurn);
```

Comment:
IUCC when a user votes, their balance gets "locked" (quotes because of comment above), and then the user has those tokens removed, changing the supply?

That would be unintuitive for me, as I'd expect the supply to remain constant if people vote. I don't think that CoinMarketCap for example will support this. Also misleading as a lot of people will think we're contracting supply.

Is there any reason the tokens need to be gone? We could just prevent then from moving rather than keeping the accounting in the supply.

Reply:
changed to lockVoteStCelo, unlockVoteStCelo and so on